package.json

```diff
@@ -31,6 +30,6 @@
   },
   "dependencies": {
     "@types/react": "^15.0.21",
```

Comment:
You can see this from the `.d.ts` file it creates in node modules. See below:
```
/// <reference types="lodash" />
import { DebounceSettings } from 'lodash';
export default function debounce(wait?: number, options?: DebounceSettings): (target: any, key: string, descriptor: PropertyDescriptor) => {
    configurable: boolean;
    get(): any;
};
```
You can see that `DebounceSettings` is being imported from `lodash` types so it needs to be a `dependency` and not a `devdependency`